shards/tests/ui-style.shs

```diff
@@ -6,49 +6,30 @@
   Once({
     GFX.DrawQueue = ui-draw-queue
     GFX.UIPass(ui-draw-queue) >> render-steps
+    "ui-nested.shs" | FS.Read | Shards.Read(AstType::Object) = ast
+    Shards.Print >= text-code
   })
   GFX.MainWindow(Title: "Shards Window" Width: 1280 Height: 720 Contents: {
     UI({
-      UI.Window(Contents: {
-        UI.Style(AnimationTime: 0.5
-          OverrideTextColor: @color(255 100 100)
-          Selection: {
-            BGFill: @color(0 255 205 255)
-            Stroke: @stroke-2
-          }
-        )
-        UI.WidgetStyle(
-          Inactive: {
-            BGFill: @color(0 0 127)
-            WeakBGFill: @color(0 0 20)
-            BGStroke: @stroke-2
-            Rounding: @f4(1.0)
-            ; Stroke: @no-stroke
-          }
-          NonInteractive: {
-            BGFill: @color(127 127 127 255)
-            Stroke: @no-stroke
-            BGStroke: @stroke-2
-          }
-          Hovered: {
-            BGFill: @color(190 190 190 255)
-            Stroke: @stroke-4
-            BGStroke: @stroke-4
-          }
-          Active: {
-            BGFill: @color(0 255 205 255)
-            Stroke: @stroke-2
-            BGStroke: @stroke-2
-          }
-          Open: {
-            BGFill: @color(255 255 255 255)
-            Stroke: @no-stroke
-            BGStroke: @stroke-2
-          }
-        )
-        "Label" | UI.Label
-        UI.Button("Button")
-        UI.TextField(text)
+      ; UI.Window(Title: "1" Contents: {
```

Comment:
Doesn't work? can you revert?

Reply:
Or make this a separate test, since seems like you hijacked this one for the UI.Shards

Reply:
eh I thought I fixed this... again git town messing things up